src/views/boards.handlebars

```diff
@@ -0,0 +1,17 @@
+{{!-- Links to the board's page --}}
+<div id="boards">
+    {{#each boards}}
+        <div class="board-link" style="background-color: {{board.boardColor}};">
+            <a href="board/{{board._id}}">{{board.boardName}}</a>
+        </div>
+    {{/each}}
+</div>
+
+{{!-- Add a new board --}}
+<form action="/boards" method="POST">
+    <label>
```

Comment:
Should this label have a for attribute or is it not needed because you put the label around the input

Reply:
Not needed